.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: java
 jdk:
   - oraclejdk7
-  - openjdk6
```

Comment:
Sure. We can replace with openjdk7. Actually I raised the issue in the dev
group yesterday. I can follow up the existing procedure to get this done.
Please advise
On Jan 25, 2015 12:01 PM, "Tom Morris" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/OpenRefine/OpenRefine/pull/934#discussion_r23505520>:
>
> > @@ -1,6 +1,5 @@
> >  language: java
> >  jdk:
> >    - oraclejdk7
> > -  - openjdk6
>
> Dropping support for an environment should be called out explicitly rather
> than hidden in the patch. We'll probably replace this with openjdk7.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenRefine/OpenRefine/pull/934/files#r23505520>.
>